source/DistanceAndDirection/ArcMapAddinDistanceAndDirection/ArcMapAddinDistanceAndDirection/ViewModels/LinesViewModel.cs

```diff
@@ -215,7 +216,10 @@ internal override void OnEnterKeyCommand(object obj)
             HasPoint2 = true;
             IGeometry geo = CreatePolyline();
             IPolyline line = geo as IPolyline;
-            AddGraphicToMap(line);
+            IDictionary<String, Double> lineAttributes = new Dictionary<String, Double>();
+            lineAttributes.Add("distance", Distance);
```

Comment:
Perhaps these strings (X, Y, distance, angle, rings, radials, etc.) should be constants in case names would need to change